src/components/content/Notification.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="notification" :class="classType">
+    <button @click="triggerClose(notification)" class="delete" aria-label="Close notification" type="button"></button>
+    {{ notification.text }}
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Notification',
+  props: [
+    'notification',
+  ],
+  data() {
+    return {
+      timer: null,
+    };
+  },
+  created() {
+    const timeout = Object.prototype.hasOwnProperty.call(this.notification, 'timeout') ? this.notification.timeout : true;
```

Comment:
I think this will do the trick:

```javascript
const timeout = this.notification.timeout !== undefined ? this.notification.timeout : true
```
It is more readable.